examples/energy-management-app/energy-management-common/include/EnergyEvseDelegateImpl.h

```diff
@@ -139,6 +141,46 @@ class EnergyEvseDelegate : public EnergyEvse::Delegate
      */
     Status StartDiagnostics() override;
 
+    /**
+     * @brief    Called when EVSE cluster receives the SetTargets command
+     */
+    Status SetTargets(
+        const DataModel::DecodableList<Structs::ChargingTargetScheduleStruct::DecodableType> & chargingTargetSchedules) override;
+
+    EvseTargetsDelegate * GetEvseTargetsDelegate() { return mEvseTargetsDelegate; }
+
+    /**
+     * @brief Delegate should implement a handler to PrepareGetTargets
+     *
+     * This needs to load any stored targets into memory and hold it until the GetTargetsFinished is
+     * called by the cluster server.
+     *
+     * @param  Reference to EvseTargetIterator class that implements the ability
```

Comment:
It says reference here, but it's a pointer to a pointer in the API. This would benefit from being updated.

Reply:
This code has been reworked and this part of the API is obsolete.